process_manager/tables.py

```diff
@@ -26,6 +27,16 @@
     "<a href=\"{% url 'process_manager:logs' record.uuid %}\">LOGS</a>"
 )
 
+header_checkbox_hyperscript = "on click set .row-checkbox.checked to my.checked"
+
+row_checkbox_hyperscript = """
+on click
+if <input.row-checkbox:not(:checked)/> is empty
```

Comment:
Why `input.row-checkbox` and not `.row-checkbox` like above?

Reply:
No good reason. I'll make them consistent.